project_resource_calendar/wizard/reservation_validation_wiz.py

```diff
@@ -0,0 +1,38 @@
+# @ 2018 Savoir-failre Linux
+# License LGPL-3.0 or Later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class DoubleBookingValidationWiz(models.TransientModel):
```

Comment:
this class looks like the DoubleTaskValidationWiz class except one field. Better to use the same class by adding the task_id field and checking if it is task or event

Reply:
Done